.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
           matrix_total: ${{ matrix.total }}
           address: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_address }}
           token: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_token }}
+          admin_token_by_role: ${{ fromJSON(steps.tflocal.outputs.workspace-outputs-json).tfe_admin_token_by_role.value }}
```

Comment:
I see a couple of issues: What is the `value` key? I'm pretty sure this object only has keys [named after each role](https://github.com/hashicorp/terraform-aws-tflocal-cloud/blob/main/variables.tf#L5) Also, I'm not sure what the behavior is when you force an object into an action input and furthermore into an env, since these things expect strings only...

Reply:
Yeah I wasn't sure either to be honest.